samples/bluetooth/cap_acceptor/README.rst

```diff
@@ -1,8 +1,8 @@
 .. zephyr:code-sample:: bluetooth_cap_acceptor
-   :name: Bluetooth: Common Audio Profile Acceptor
+   :name: Common Audio Profile Acceptor
```

Comment:
What do you prefer: 

1. Common Audio Profile Acceptor
2. Common Audio Profile (CAP) Acceptor
3. CAP Acceptor

Just in case we want to be consistent with the Audio samples later

Reply:
2. is probably best as this makes it easier to capture all the various types/combination of search queries users might perform

I thought someone said they would address naming in a follow-up PR though? :P